site/docs/be/docker.md

```diff
@@ -0,0 +1,4 @@
+---
+layout: redirect
+redirect: docs/be/docker.html
```

Comment:
AFAICT, if we have any links pointing at .mds, they are just wrong and should just 404. 

Does this actually work?  You're not usually going to be in a place where a relative link to docs/be/docker.html actually resolves to /versions/master/docs/be/docker.html.